bin/dbunloader.py

```diff
@@ -114,6 +119,24 @@
     interval = cp.get('unloader', 'interval')
 
     unloader = DbUnloader(db, unload_dir, include_vos, exclude_vos, local_jobs, withhold_dns)
+
+    try:
```

Comment:
Should should probably factored out to a function that we can test.

Reply:
It's just consistent with the other option parsing code.
Happy to open a separate PR that factors all of them out though.

Reply:
Fair enough.